src/test/java/EOorg/EOeolang/EOtxt/EOregexEOmatchTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```

Comment:
@levBagryansky I think here should be `# 2016-2022 Objectionary.com`

Reply:
@Graur , 
checkstyle requires  Copyright (c) 2022 Eugene Darashkevich in java files

Reply:
@Graur I also decided do the same in eo files